recipes-core/base-files/base-files_%.bbappend

```diff
@@ -6,24 +6,35 @@ do_install:append() {
 
     # add Aos partitions
     echo "# Aos partitions">> ${D}${sysconfdir}/fstab
-    echo "/dev/aosvg/workdirs  /var/aos/workdirs    ext4 \
-        defaults,nofail,noatime,x-systemd.automount  0  0" >> ${D}${sysconfdir}/fstab
+    echo '/dev/aosvg/workdirs  /var/aos/workdirs   ext4 defaults,nofail,noatime,x-systemd.automount'\
+${@bb.utils.contains('DISTRO_FEATURES', 'selinux', ',context=system_u:object_r:aos_var_run_t:s0', '', d)} '0 0' \
+   >> ${D}/${sysconfdir}/fstab
```

Comment:
Commit message: `Add selinux context to fstab`

Reply:
Done